jenkins/image_building/build-image.sh

```diff
@@ -63,7 +63,7 @@ fi
 
 export HOSTNAME="${img_name}"
 
-disk-image-create --no-tmpfs -a amd64 -o "${img_name}".qcow2 "${IMAGE_OS}"-"${IMAGE_TYPE}" block-device-efi
+disk-image-create --no-tmpfs -a amd64 -o "${img_name}" -t raw "${IMAGE_OS}"-"${IMAGE_TYPE}" block-device-efi
```

Comment:
`img_name` does not have any suffix, as its defined on L57 or L61. Does the command automatically give it `.raw` when `-t raw` is given?

Reply:
Yes

Reply:
Not a fan of implicit suffix. If we add it here, does it duplicate it? If not, then absolutely we want it here explicitly.

Reply:
I have a feeling we'll trip over this later, if we don't add it. Implicit assumptions tend to screw you up sooner or later. It would be great if you could try it with explicit suffix.

Reply:
Ok lets put a hold then. I will push and test it later. We are not in a hurry with this .

/hold

Reply:
Added and its working.